app/User.php

```diff
@@ -26,4 +26,35 @@ class User extends Authenticatable
     protected $hidden = [
         'password', 'remember_token',
     ];
+
+    /**
+    * Method for middleware user
+    *
+    * @return boolean
+    */
+    public function isUser()
```

Comment:
[rank B] Do we manage system roles in database? If it's simple, we need define it as constants in config files. I think meaningless when we use method like that.